Public/Src/Tools/Xldb.Analyzer/XldbAnalyzer.cs

```diff
@@ -103,8 +104,8 @@ public int AnalyzeEventStats()
         {
             if (m_commandLineOptions.ContainsKey("/h"))
             {
-                Console.WriteLine("\nEvent Stats Analyzer");
-                Console.WriteLine("Generates stats on the aggregate size and count of execution log events, but uses the RocksDB database as the source of truth");
+                Console.WriteLine("\nDB Stats Analyzer");
+                Console.WriteLine("Generates stats on the aggregate size and count of different items stored in the DB, but uses the RocksDB database as the source of truth");

```

Comment:
nit: it does not 'generate' stats, it dumps the metadata you stored in the db while you were creating it